apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/step-based-flow.tsx

```diff
@@ -242,6 +247,20 @@ export const StepBasedFlow: FunctionComponent<AuthenticationFlowPropsInterface>
         }
     }, [ authenticationSteps ]);
 
+    /**
+     * Change button disable state when authentication Steps are changed.
+     */
+    useEffect(() => {
+
+        const noOptionsSelected = authenticationSteps.length === 1 && authenticationSteps[ 0 ].options?.length === 0;
```

Comment:
```suggestion
        const noOptionsSelected: boolean = authenticationSteps.length === 1 && authenticationSteps[ 0 ].options?.length === 0;
```

Reply:
fixed in 7402005b3cafad3dd06a3f785122cb2d30e11cf4

Reply:
fixed in 7402005b3cafad3dd06a3f785122cb2d30e11cf4